server/controllers/auth.js

```diff
@@ -42,21 +51,29 @@ const register = (req, res) => {
 const login = (req, res) => {
     const username = req.body.username;
     const password = req.body.password;
-    // TODO: need to hash the password
-    db.query(`SELECT * FROM guessthelocation.user_profile WHERE username='${username}' AND password='${password}';`, function (error, results, fields) {
+    console.log("we received...");
+    console.log(username);
+    console.log(password);
+    // need to hash the password
+    db.query(`SELECT * FROM GuessTheLocation.user_profile WHERE username='${username}' AND password='${password}';`, function (error, results, fields) {
         if (error) {
-            return res.status(500).json({ error: "Internal Server Error." });
+            console.log("here???");
+            return res.status(400).json({ error: "Error" });
         }
-        if(results.length === 1) {
+        if(results.length != 0) {
             console.log("success, starting session now");
             // create cookie here
+            const sesh = req.session;
+            console.log(sesh);
+            sesh.username = username;
+            sesh.password = password;
 
             // login successful
             return res.status(200).json({ message: "Login Successful!" });;
             
         } else {
-            console.log("WRONG PASSWORD!!!!!!")
-            return res.status(400).json({ error: "Password Incorrect or multiple user entries." });
+            console.log("Password is wrong, or the account hasn't been registered!")
```

Comment:
please move the console.log message to the json that you return

Reply:
For this one I think it would be better to keep it, since in the future we might need to add pop-up message in the website. This consol.log here can serve as a hint for error type.